sdk/textanalytics/tests.yml

```diff
@@ -16,9 +16,9 @@ stages:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_TEXT_ANALYTICS_API_KEY: $(java-textanalytics-test-api-key)
         AZURE_TEXT_ANALYTICS_ENDPOINT: $(java-textanalytics-test-ppe-endpoint-string)
-        AZURE_TEXT_ANALYTICS_CUSTOM_SINGLE_CLASSIFICATION_PROJECT_NAME: $(java-single-category-classify-project-name)
-        AZURE_TEXT_ANALYTICS_CUSTOM_SINGLE_CLASSIFICATION_DEPLOYMENT_NAME: $(java-single-category-classify-deployment-name)
-        AZURE_TEXT_ANALYTICS_CUSTOM_MULTI_CLASSIFICATION_PROJECT_NAME: $(java-multi-category-classify-project-name)
-        AZURE_TEXT_ANALYTICS_CUSTOM_MULTI_CLASSIFICATION_DEPLOYMENT_NAME: $(java-multi-category-classify-deployment-name)
-        AZURE_TEXT_ANALYTICS_CUSTOM_ENTITIES_PROJECT_NAME: $(java-recognize-custom-entities-project-name)
-        AZURE_TEXT_ANALYTICS_CUSTOM_ENTITIES_DEPLOYMENT_NAME: $(java-recognize-custom-entities-deployment-name)
+        AZURE_TEXT_ANALYTICS_CUSTOM_SINGLE_CLASSIFICATION_PROJECT_NAME: $(js-single-category-classify-project-name)
```

Comment:
does this reference the same values as the java one? or do you need to change the endpoint/key as well?

Reply:
Yes. They are referencing the same values.
It is the same endpoint that all languages share. I was doing the Live-test testing to change to use the JS variable but actually java variable of `java-recognize-custom-entities-deployment-name` copied from `js-recognize-custom-entities-deployment-name`

Reply:
revert the changes. It doesn't make the CI pass